server.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+
+const app = express();
+
+
+
+app.use(express.static('build'));
+
+app.listen(8080 , function () {
+    console.log('server is running on port 8080')
+  
+});
```

Comment:
Let's not use this server coupled with client code

Try install "http-server" for a simple server

@Kratika0907